packages/experiments-realm/blog-app.gts

```diff
@@ -230,10 +234,8 @@ interface CardAdminViewSignature {
 }
 class BlogAdminData extends GlimmerComponent<CardAdminViewSignature> {
   <template>
-    <div class='blog-admin' ...attributes>
-      {{#if this.resource.cardError}}
-        Error: Cannot render card "{{@cardId}}"
```

Comment:
Is it okay for there to be no error handling now? Or maybe it’s handled elsewhere in a way I’m not seeing?

Reply:
Thanks, added this back with a shorter message